### PR TITLE
Add preliminary documentation for okta input

### LINKED ISSUE #859: Add documenation for Okta Log Events input
## Context

Provided general documentation on how the Okta Log Events input plugin works

## Expected Content

No documentation exists for the Okta Log Events input plugin

## Your Environment

* Graylog Version: 3.3
Screen Shot 2020-04-27 at 3 56 10 PM

### PR DESCRIPTION
**Pull request by cobaltclaudia**
closes #859

### CODE REVIEW

File: pages/integrations/inputs/okta_input.rst
Comment (waab76) [inline pages/integrations/inputs/okta_input.rst]:
```suggestion
.. attention:: This is a Graylog Enterprise feature and is only available since Graylog version 3.3.  A valid Graylog Enterprise license is required.
```

File: pages/integrations/inputs/okta_input.rst
Comment (waab76) [inline pages/integrations/inputs/okta_input.rst]:
```suggestion
This input allows Graylog to pull event logs from your Okta domain.
```

File: pages/integrations/inputs/okta_input.rst
Comment (waab76) [inline pages/integrations/inputs/okta_input.rst]:
```suggestion
For this input plugin to function as expected, the following items must be supplied in the input configuration:
```

File: pages/integrations/inputs/okta_input.rst
Comment (waab76) [inline pages/integrations/inputs/okta_input.rst]:
```suggestion
    Your Okta Domain (also known as your Okta URL).  You can copy your domain from the Okta Developer Console.
```

Comment (waab76) [reply pages/integrations/inputs/okta_input.rst]: Also consider adding a link to https://developer.okta.com/docs/guides/find-your-domain/overview/

File: pages/integrations/inputs/okta_input.rst
Comment (waab76) [inline pages/integrations/inputs/okta_input.rst]:
```suggestion
    The token which will be used to authenticate Graylog's requests to Okta.  You can create an API token on the Okta Developer Console.
```

Also consider adding a link to https://developer.okta.com/docs/guides/create-an-api-token/-/overview/

File: pages/integrations/inputs/okta_input.rst
Comment (waab76) [inline pages/integrations/inputs/okta_input.rst]: Need to add information about the Polling Interval.  Determines how often we will request new data from Okta.  Cannot be smaller than 5 seconds.

File: pages/integrations/inputs/okta_input.rst
Comment (waab76) [inline pages/integrations/inputs/okta_input.rst]: Should note that the Keyword filter is optional.

If it is provided, cannot have more than 10 keywords (space-separated) and keywords cannot be more than 40 characters long.

**Review by waab76 — CHANGES_REQUESTED** [review]

**Review by waab76 — DISMISSED** [review]
LGTM.  I'll wait for @lingpri to have a look before merging.

**Review by jalogisch — COMMENTED** [review]
He,

I like this documentation, but we should include some "starting" sentences about WHAT kind of logs this integration gets from what okta product. That someone can identify quickly if that is what he is looking for. 

From the given information I could not decide if this is something I need or not.

File: pages/integrations/inputs/okta_input.rst
Comment (lingpri) [inline pages/integrations/inputs/okta_input.rst:26]: Please add a link, to the api link. 
https://developer.okta.com/docs/reference/api/system-log/ 
The document looks good.

**Review by lingpri — COMMENTED** [review]
LGTM. Also, its good for someone outside our team , understand the docs. I will leave it to @jalogisch to merge it.

**Review by lingpri — DISMISSED** [review]

File: pages/integrations/inputs/okta_input.rst
Comment (lingpri) [inline pages/integrations/inputs/okta_input.rst]: thanks for adding this line. it will ensure, they don't check the global checkbox, we didn't find a way to remove the `isGlobal` check box yet.

File: pages/integrations/inputs/okta_input.rst
Comment (jalogisch) [inline pages/integrations/inputs/okta_input.rst]: > The Okta System Log records system events related to your organization in order to provide an audit trail that can be used to understand platform activity and to diagnose problems.

The above is taken from the referenced page - maybe it would make sense to integrate that too? 

Why someone should install this plugin? What benefit comes out of that? Not in marketing speak - from a technical point of view.

File: pages/integrations/inputs/okta_input.rst
Comment (jalogisch) [reply pages/integrations/inputs/okta_input.rst]: This is better than nothing. But this will be something that came up regular. Will it be able to prevent that more than one input is running? Could we implement something in some way?

**Review by jalogisch — CHANGES_REQUESTED** [review]
It looks good - but IMHO need some additional information what you get into Graylog with this plugin and why you want this.

File: pages/integrations/inputs/okta_input.rst
Comment (lingpri) [reply pages/integrations/inputs/okta_input.rst]: **What is Okta ?***

Helps organization with 2 things. 

okta_solutions


**Why graylog Okta Plugin ?** 
* launch the `okta Log Events` plugin to ingest okta syslog and analyze them. 
* [tell stories, that are insightful, inspiring , innovate,change the way you think, with a custom dashboard.](https://github.com/Graylog2/documentation/blob/3.2/pages/dashboards.rst#thats-it)
* set up [alerts](https://github.com/Graylog2/documentation/blob/3.2/pages/alerts.rst#id2) with a combination of fields, saving millions of dollars on down time events, save the earth.  
* pay attention to the odd-balls in your data.
* refer @waab76 notes from the all hans demo.

File: pages/integrations/inputs/okta_input.rst
Comment (cobaltclaudia) [reply pages/integrations/inputs/okta_input.rst]: updated the verbiage. if global is selected, then the input will default to only running on the master node

**Review by jalogisch — APPROVED** [review]
👍 LGTM Thank you

**Pull request merged**
